setup.py

```diff
@@ -11,4 +12,4 @@
       url='http://www.arcoslab.org/',
       package_dir={'pyrovito': ''},
       packages=['pyrovito'],
```

Comment:
This will only package pyrovito, not your new script. I suggest changing to the `src/` directory structure:
```bash

├── COPYING
├── Makefile
├── README.md
├── setup.py
└── src
   └── pyrovito
      ├── __init__.py
      ├── pyrovito_utils.py
      └── scripts
         ├── __init__.py
         ├── pyrovito
         └── vis_sink.py
```
And adjust the `setup.py` accordingly